cluster_management/src/main/java/com/pinterest/rocksplicator/MasterSlaveStateModelFactory.java

```diff
@@ -297,6 +301,7 @@ public void onBecomeMasterFromSlave(Message message, NotificationContext context
             e);
       } finally {
         leaderEventsCollector.commit();
```

Comment:
same here

Reply:
There's a log statement just before, at the end of the try block, so that it was unnecessary here.